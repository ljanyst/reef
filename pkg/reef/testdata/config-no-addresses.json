{
  "Web": {
    "Https": {
      "Enabled": true,
      "Cert": "cert.pem",
      "Key": "key.pem",
      "Chain": "chain.pem"
    }
  },
  "Backend": {
    "DatabaseDirectory": "data"
  }
}
