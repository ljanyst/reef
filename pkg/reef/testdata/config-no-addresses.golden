&{Web:{BindAddresses:[{Host:localhost Port:7651 IsHttps:false}] Https:{Cert:cert.pem Key:key.pem} EnableAuth:false HtpasswdFile:} Backend:{DatabaseDirectory:data}}
